qutip/solver/ode/explicit_rk.pyx

```diff
@@ -0,0 +1,283 @@
+#cython: language_level=3
+#cython: boundscheck=False, wraparound=False, initializedcheck=False, nonecheck=False
+from .wrapper cimport QtOdeData, QtOdeFuncWrapper
+from .wrapper import qtodedata
+cimport cython
+cimport numpy as cnp
+cdef extern from *:
+    void *PyDataMem_NEW(size_t size)
+    void *PyDataMem_NEW_ZEROED(size_t size, size_t elsize)
+    void PyDataMem_FREE(void *ptr)
+
+cnp.import_array()
+
+
+cdef class Explicit_RungeKutta:
+    def __cinit__(Explicit_RungeKutta self):
+        self.rk_step = 1
+        self.rk_extra_step = 1
+        self.order = 1
+        self.denseout_order = 1
+        self.can_interpolate = False
```

Comment:
This is for other Runge-Kutta method, if we decide to add the traditional 4th order, it would have `can_interpolate=False`. If 
`Explicit_RungeKutta` object is used directly, it implement euler method and has `can_interpolate=False`.